generate.go

```diff
@@ -204,7 +211,12 @@ type enumTypeMap map[string]enumValues
 
 // GenerateProtobufDefinition generates a .proto file from a list of structs via reflection.
 // fieldNamer is a function that maps ProtoField types to generated protobuf field names.
-func GenerateProtobufDefinition(w io.Writer, types []interface{}, enumMap EnumMap, renamer GeneratorNamer) error {
+func GenerateProtobufDefinition(w io.Writer, types []interface{}, enumMap EnumMap, renamer GeneratorNamer) (err error) {
+	defer func() {
+		if e := recover(); e != nil {
```

Comment:
YES! Like this! One in the public function, and it returns errors instead of printing them!